client/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@finos/git-proxy": "file:.."
+  }
+}
```

Comment:
Why are we making this change?

Reply:
I installed dependencies for this project using the commands  `npm run clientinstall` ( which installed the above dependency ) and `npm install`. This dependency is not relatable for branch protection. 

Branch protection can be modified by making the above changes in branch settings which I have shown in the screenshots.